koku/masu/celery/tasks.py

```diff
@@ -228,15 +75,29 @@ def upload_normalized_data():
     accounts, _ = Orchestrator.get_accounts()
 
     # Deduplicate schema_name since accounts may have the same schema_name but different providers
```

Comment:
This comment is no longer valid. Accounts is built from the list of providers: https://github.com/project-koku/koku/blob/master/koku/masu/external/accounts/db/cur_accounts_db.py#L76 so we don't need deduplication. 

You could remove
```
    schema_providers = set(
        (account['schema_name'], account['provider_uuid']) for account in accounts
    )
```
and iterate through the accounts. This save only a marginal amount of time so I completely understand if you want to leave as is.

Reply:
TIL! I'll prune this out with the next push.